### PR TITLE
[ADLPS] CRB PV release update

### PR DESCRIPTION
**Pull request by koktong-ong**
Update BoardID and PlatformID for CRB board
Added ddi config for CRB board
Update FSPM UPD due to common value across all sku

Signed-off-by: Ong Kok Tong <kok.tong.ong@intel.com>

### CODE REVIEW

File: Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h [hunk 1]
```diff
@@ -25,8 +25,8 @@ Defines Platform BoardIds
 #define BoardIdAdlPSDdr5Rvp                           0x0B
 #define PLATFORM_ID_ADL_PS_DDR5_RVP                   0x0B
 
-#define BoardIdAdlPSDdr5Crb                           0x0D // MiniITX is CRB
-#define PLATFORM_ID_ADL_PS_DDR5_CRB                   0x0D
+#define BoardIdAdlPSDdr5Crb                           0x0C // MiniITX is CRB
```
Comment (gdong1) [inline Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h:28]: what's the value read from SMBUS for PS board? Often this value would not change once we know the value, not sure  why changes BoardIdAdlPSDdr5Crb here.

Comment (koktong-ong) [reply Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h:28]: the value read from SMBUS is 0xD but due to the offset workaround in line 125 in EcSupport.c
*BoardId = (SmbusInputPort1Info.InputPort1Fields.BoardId - ADL_SMBUS_BOARDID_OFFSET);
The reason with this workaround is we seeing same conflict (same BoardID) between ADLN CRB and ADLPS RVP
https://github.com/slimbootloader/slimbootloader/commit/0dc5b1da43db0232098b15ecb55a090903b874ee

Comment (koktong-ong) [reply Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h]: Updated the patch as per discussion

**Comment by koktong-ong** [comment]
@gdong1 
Updated the patch as per discussion

**Review by gdong1 — APPROVED** [review]

**Pull request merged**